### PR TITLE
Switch back to plain `npm install`.

### LINKED ISSUE #1507: Move back to npm --prefix instead of using -C to install components
See https://ci.appveyor.com/project/XhmikosR/lighthouse/build/59/job/uis3t57c4558yf18 for an example

What does the `-C` npm flag do exactly?

### PR DESCRIPTION
**Pull request by XhmikosR**
`-C` didn't work on Windows.

Fixes #1507.

### CODE REVIEW

**Comment by paulirish** [comment]
instead of 
`npm install -C ./lighthouse-cli`

how about just
`npm install --prefix ./lighthouse-cli`
?

is that good enough, or does it need the destination path explicit as well?

**Comment by XhmikosR** [comment]
Unfortunately, it does for me on Windows.

Please check my comment in https://github.com/GoogleChrome/lighthouse/issues/1507#issuecomment-274967215

**Comment by ebidel** [comment]
@paulirish we can do `cd ./lighthouse-cli && npm install` if you think it's too tricky. Anyone reading that link would instantly know what it does and wouldn't have to read up on `--prefix` (me). Also no need for pushd/popd, yea?

Either way, it would be nice to be consistent about the method in our main package.

**Comment by XhmikosR** [comment]
Yeah, `cd` / `pushd` /`popd` would be better IMO.

**Comment by XhmikosR** [comment]
I added the solution I personally prefer. Let me know which one you want.

**Comment by paulirish** [comment]
sure. wfm!

**Comment by XhmikosR** [comment]
You want to squash and rename the commit? I can do it tomorrow if you can't.

On a side note, the build* scripts work with `-C` but they create an empty
etc folder in the subfolders. This is probably a bug in npm. We could
change those to do the same with cd for consistency.

File: package.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
   },
   "scripts": {
     "install-all": "npm run install-cli && npm run install-extension && npm run install-viewer",
-    "install-cli": "npm install -C ./lighthouse-cli",
-    "install-extension": "npm install -C ./lighthouse-extension",
-    "install-viewer": "npm install -C ./lighthouse-viewer",
+    "install-cli": "cd ./lighthouse-cli && npm install && cd ..",
```
Comment (ebidel) [inline package.json]: you shouldnt need the `&& cd .`.

**Comment by XhmikosR** [comment]
I'll test this tomorrow and change it if it works.

**Comment by ebidel** [comment]
SGTM. Agreed we should do the `build` scripts too.

**Comment by XhmikosR** [comment]
@ebidel: done. Feel free to rename the PR or change the commit message to something better.

**Pull request merged**